### PR TITLE
Add Loading State for Update and Change Placeholder for Search Filter

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4569

search-updates

### CODE REVIEW

File: components/UpdatesWithData.js [hunk 1]
```diff
@@ -77,13 +78,16 @@ class UpdatesWithData extends React.Component {
...
         <UpdateFilters values={query} onChange={onChange} />
         <Box mt={4} mb={5}>
-          <Updates
-            collective={collective}
-            updates={updates}
-            editable={!compact}
-            fetchMore={this.props.fetchMore}
-            LoggedInUser={LoggedInUser}
-          />
+          {data.loading && <Loading />}
```
Comment (Betree) [inline components/UpdatesWithData.js:81]: :ice_cream: It's purely a style preference (no requested change here), but I like to use ternary conditions for these cases. I find them very clear to read, and it prevents checking multiple time the same thing.

```jsx
{data.loading ? (
  <Loading />
) : (
  <Component />
)
```

**Review by Betree — APPROVED** [review]
Thanks for this follow-up, it's looking good!

File: components/UpdatesWithData.js (hunk 1 above)
Comment (SudharakaP) [reply components/UpdatesWithData.js:81]: Thanks, let me keep this in mind. 👍🏽 I've made the change.

**Comment by SudharakaP** [comment]
> Thanks for this follow-up, it's looking good!

You are welcome. Thanks for all the reviews. 🙏🏽

**Pull request merged**